wg-ml-experience/charter.md

```diff
@@ -0,0 +1,63 @@
+# WG ML Experience
+
+This charter adheres to the conventions, roles, and organisation management outlined in [wg-governance] for the Working Group "ML Experience".
+
+## Scope
+
+The ML Experience Working Group focuses on developing, maintaining, and improving tools and extensions that support data science and machine learning practitioners workflows within Kubeflow. The group is dedicated to delivering a high-level, seamless experience integrated with the IDE of choice across multiple Kubeflow components.
+ 
+### In scope
+ 
+#### Code, Binaries, and Other relevant assets
+
+1. Development of Kubeflow JupyterLab extensions that provide simple abstractions and UX to interact with the most common Kubeflow components (e.g., pipelines, hyperparameter tuning) and shorten the time to value for practitioners comfortable with Jupyter. These extensions will focus on the most used Kubeflow components, such as:
+    - Pipelines;
+    - Training Operator & Katib;
+    - Model Registry;
+    - Model Serving (KServe);
+    - Feast.
+
+2. Promote the reusability of UI components from other Kubeflow UIs into the IDE (e.g., rendering a pipeline graph inside the JupyterLab environment) by establishing a shared contract between the IDE WG and the wider Kubeflow community. 
+
+3. Develop a Python SDK to simplify operationalization across Kubeflow components and provide a “one-stop-shop” for practitioners who want easy access to Kubeflow services. The SDK also provides the groundwork for the IDE extension automation and workflows.
+    - Create a single installation and configuration layer for users interacting programmatically with the Kubeflow ecosystem via SDKs.
+    - The “common” SDK is not meant to replace individual components’ SDKs but rather to offer a unified access layer to simplify dependency management and shared configuration (like authorization).
+
+#### Guiding Principles
```

Comment:
What about set of standards to make sure Kubeflow UIs follow the same design language and architecture principle. Like what @StefanoFioravanzo and @kimwnasptd discussed before here: https://github.com/kubeflow/community/issues/780

Reply:
Don't you think this implicity covers it?
>- Synergy among Kubeflow Working Groups: Collaborate with other WG to promote reusability of UI components from other Kubeflow UIs to create a single UX between Jupyter IDE and Kubeflow Central Dashboard;

Reply:
Shell we also say something about Kubeflow UI Design Language or your think that is clear from this statement ?

Reply:
You are right, I just made an statement covering this in the FUP PR.